apps/omg_watcher_rpc/lib/web/views/transaction.ex

```diff
@@ -61,7 +61,9 @@ defmodule OMG.WatcherRPC.Web.View.Transaction do
     outputs = Map.fetch!(transaction, :outputs)
 
     transaction
-    |> Map.take([:txindex, :block, :txhash, :metadata])
+    |> Map.take([:txindex, :txhash, :block, :inputs, :outputs, :txbytes, :metadata])
```

Comment:
When/where available it would be nice to have the UTXO root chain txn hash via `child_chain_utxohash` joined through to the ethevents table. My original commit was WIP and was just returning `child_chain_utxohash` instead of `rootchain_txhash`. Did we decide we don't want this data anymore displaying deposits, challenges, exists, etc?

I don't see any performance considerations to worry about, if we think that might be an issue...

Reply:
No issues with that, but shouldn't that go in a separate PR since it's not required for this PR/user story?

Reply:
Ah maybe it's the PR title. Updated.